.github/workflows/tests.yaml

```diff
@@ -22,11 +22,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
-      - name: run app locally
-        uses: ./.github/actions/local-app-run
```

Comment:
If you want to try removing this from the jobs below too, I don't think it's needed

Reply:
The local-app-run deploys the DB, so it's needed for the schema and pgtap tests. It should be refactored, but if we can leave it for another ticket since in the meantime sonarcloud isn't being run on other PRs